src/controllers/user.js

```diff
@@ -176,6 +209,67 @@ const User = {
 
     return res.status(200).send({ imgURL });
   },
+
+  verify: async (req, res, next) => {
+    const { id } = req.params;
+    const { verificationCode } = req.body;
+    let result;
+
+    try {
+      result = await UserModel.findOneAndUpdate({ _id: id, verificationCode },
+        { verified: true }).exec();
+    } catch (err) {
+      return next(new APIError());
+    }
+
+    if (!result) {
+      return next(new APIError(
+        'User could not be found',
+        `User with id ${id} was not found`,
+        404,
+        `users/${id}/verify`,
+      ));
+    }
+
+    return res.status(200).send(result.toJSON());
+  },
+
+  emailPasswordRecovery: async (req, res, next) => {
+    const { id } = req.params;
```

Comment:
If someone is resetting their password, they probably cant login. As such, we wont have a way to get the id of the user. We should ask them for their email instead.

Reply:
This is not the method to send the verification email, it's the method that'll be executed once the user hits the verification link, it will have the ID by then

Reply:
Actually disregard my previous comment